### PR TITLE
Create Project and Projectlist api integration done

### PR DESCRIPTION
**Pull request by aghilodessey**
-->Create Project and Projectlist api integration done
wed1
wed2

### CODE REVIEW

File: client/src/pages/projects/hooks/projects.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { useToast } from "@chakra-ui/react";
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+
+import axios from "axios";
+
+const fetchProjectList = async () => {
+  const url = "http://logger.odesseylabs.com/api/project";
```
Comment (nihiannd) [inline client/src/pages/projects/hooks/projects.hooks.ts]: Move baseurl to the .env file and call where ever needed

File: client/src/pages/projects/hooks/projects.hooks.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    sort: {
+      sortKey: "projectId",
+      sortOrder: "ASC",
+    },
+    limit: 20,
+    offset: 0,
+  };
+
+  const headers = {
+    Authorization: "Bearer NIBBLES_ODESSEY",
```
Comment (nihiannd) [inline client/src/pages/projects/hooks/projects.hooks.ts]: Move to .env file

File: client/src/pages/projects/hooks/projects.hooks.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  return useQuery(["projectList"], fetchProjectList);
+};
+
+export const useCreateProjectForm = (
+  onSuccessCallBack?: VoidFunction,
+  onErrorCallBack?: VoidFunction
+) => {
+  const toast = useToast();
+
+  const CREATE_PROJECT = `http://logger.odesseylabs.com/api/project/create`;
```
Comment (nihiannd) [inline client/src/pages/projects/hooks/projects.hooks.ts]: move baseurl to .env

File: client/src/pages/projects/hooks/projects.hooks.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  const headers = {
+    Authorization: "Bearer NIBBLES_ODESSEY",
```
Comment (nihiannd) [inline client/src/pages/projects/hooks/projects.hooks.ts]: move to .env

**Review by nihiannd — CHANGES_REQUESTED** [review]

File: client/src/pages/projects/hooks/projects.hooks.ts (hunk 1 above)
Comment (aghilodessey) [reply client/src/pages/projects/hooks/projects.hooks.ts]: Done

File: client/src/pages/projects/hooks/projects.hooks.ts (hunk 2 above)
Comment (aghilodessey) [reply client/src/pages/projects/hooks/projects.hooks.ts]: Done

File: client/src/pages/projects/hooks/projects.hooks.ts (hunk 3 above)
Comment (aghilodessey) [reply client/src/pages/projects/hooks/projects.hooks.ts]: Done

File: client/src/pages/projects/hooks/projects.hooks.ts (hunk 4 above)
Comment (aghilodessey) [reply client/src/pages/projects/hooks/projects.hooks.ts]: Done

**Review by nihiannd — APPROVED** [review]

**Pull request merged**